web/src/pages/Home/TopJurors/JurorCard/DesktopCard.tsx

```diff
@@ -5,37 +5,28 @@ import Rank from "./Rank";
 import JurorTitle from "./JurorTitle";
 import Rewards from "./Rewards";
 import Coherency from "./Coherency";
-import HowItWorks from "./HowItWorks";
+import JurorLevel from "./JurorLevel";
 
 const Container = styled.div`
   display: none;
-
-  justify-content: space-between;
-  flex-wrap: wrap;
   width: 100%;
   background-color: ${({ theme }) => theme.whiteBackground};
   border: 1px solid ${({ theme }) => theme.stroke};
   border-top: none;
   align-items: center;
   padding: 15.55px 32px;
 
-  label {
-    font-size: 16px;
-  }
-
   ${landscapeStyle(
     () => css`
-      display: flex;
+      display: grid;
+      grid-template-columns:
+        min-content repeat(2, calc(80px + (200 - 60) * (min(max(100vw, 375px), 1250px) - 375px) / 875))
+        min-content auto;
```

Comment:
Let's try and center total rewards and coherent votes
![image](https://github.com/kleros/kleros-v2/assets/80388279/4d4a3b2d-12bd-4a96-b4d7-67f13ccd6875)